Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -169,6 +169,10 @@ private void SetAsset(InputActionAsset asset, string actionToSelect = null, stri
                 {
                     var assetPath = AssetDatabase.GUIDToAssetPath(m_AssetGUID);
                     var asset = AssetDatabase.LoadAssetAtPath<InputActionAsset>(assetPath);
+
+                    if (asset == null)
```

Comment:
Hmmm is throwing the right thing to do here? Check the other places where we have fallbacks, we typically Debug.LogError + call Close() then return. Otherwise it will leak memory and maybe also relates to Paulius error? Or maybe we are good here since coming back from domain reload and all we have are the serialised objects. But at least m_AssetForEditing would leak.

Reply:
The Close() would in turn trigger Destroy which would cleanup memory, destroy UI etc.

Reply:
> Hmmm is throwing the right thing to do here? Check the other places where we have fallbacks, we typically Debug.LogError + call Close() then return. Otherwise it will leak memory and maybe also relates to Paulius error? Or maybe we are good here since coming back from domain reload and all we have are the serialised objects. But at least m_AssetForEditing would leak.

This code is enclosed in a `try...catch` block that handles exceptions as you stated: logs an error and calls `Close()` on the window. I verified this does indeed happen when reproing this scenario and also noted this pattern is used elsewhere in the source file.

So, all that to say, I think this is good :)

Reply:
Thanks @timkeo , realised I overlooked that in this review but realised when I checked out the branch. I also think this is all good. Will approve this now. Thanks for the collaboration on this PR.